src/Request/RequestHandler.php

```diff
@@ -197,7 +213,12 @@ public function handleRequest(string $method, string $uri, array $options, array
             $this->handleException($exception);
         }
 
-        return json_decode($response->getBody());
+        $contents = $response->getBody()->getContents();
+
+        if ($contentMethod) {
```

Comment:
Can you please give some more information about this?

Maybe an example of when it won't be a json_decode can help.